glide.yaml

```diff
@@ -71,6 +71,8 @@ import:
 - package: honnef.co/go/tools
   subpackages:
   - cmd/staticcheck
+- package: github.com/gogo/status
```

Comment:
cc @robbertvanginkel just in case this is a step in the wrong direction in his worldview.

Reply:
Shouldn't be. I did a deep dive into grpc-go and found that gogo/status is compatible. If we're going to continue using gogo as our proto compiler, I think we should be using gogo everywhere. We should also be doing this in v2.